contrib/completion/zsh/_docker

```diff
@@ -541,6 +561,7 @@ __docker_container_subcommand() {
         "($help)*--group=[Set one or more supplementary user groups for the container]:group:_groups"
         "($help -h --hostname)"{-h=,--hostname=}"[Container host name]:hostname:_hosts"
         "($help -i --interactive)"{-i,--interactive}"[Keep stdin open even if not attached]"
+        "($help)--init[Run an init inside the container that forwards signals and reaps processes]" \
```

Comment:
This trailing backslash should be removed; I think there's a fix in another PR, but that also has other changes, so we should make sure that either that one, or a fix for just this trailing backslash makes it into 1.13.1